proto/osmosis/concentrated-liquidity/genesis.proto

```diff
@@ -46,26 +44,43 @@ message PoolData {
   // incentive records to be set
   repeated IncentiveRecord incentive_records = 5
       [ (gogoproto.nullable) = false ];
+
+  repeated PositionData position_data = 6 [ (gogoproto.nullable) = false ];
 }
 
 message PositionData {
-  Position position = 1;
+  PositionWithoutPoolId position = 1;
   uint64 lock_id = 2 [ (gogoproto.moretags) = "yaml:\"lock_id\"" ];
   osmosis.accum.v1beta1.Record spread_reward_accum_record = 3
       [ (gogoproto.nullable) = false ];
   repeated osmosis.accum.v1beta1.Record uptime_accum_records = 4
       [ (gogoproto.nullable) = false ];
 }
 
+message PositionWithoutPoolId {
+  uint64 position_id = 1 [ (gogoproto.moretags) = "yaml:\"position_id\"" ];
+  string address = 2 [ (gogoproto.moretags) = "yaml:\"address\"" ];
+  int64 lower_tick = 3;
+  int64 upper_tick = 4;
+  google.protobuf.Timestamp join_time = 5 [
+    (gogoproto.nullable) = false,
+    (gogoproto.stdtime) = true,
+    (gogoproto.moretags) = "yaml:\"join_time\""
+  ];
+  string liquidity = 6 [
+    (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
+    (gogoproto.moretags) = "yaml:\"liquidity\"",
+    (gogoproto.nullable) = false
+  ];
+}
+
 // GenesisState defines the concentrated liquidity module's genesis state.
 message GenesisState {
   // params are all the parameters of the module
   Params params = 1 [ (gogoproto.nullable) = false ];
   // pool data containining serialized pool struct and ticks.
   repeated PoolData pool_data = 2 [ (gogoproto.nullable) = false ];
 
-  repeated PositionData position_data = 3 [ (gogoproto.nullable) = false ];
```

Comment:
err, where is position data now? Its not in pool data?

Reply:
ah PoolData was changed. Should we change that to "genesis PoolData" ?

Reply:
Done in https://github.com/osmosis-labs/osmosis/pull/5453/commits/971283a67849bdd6b29926104620021c38ec4ca9